alpine/src/main/scala/actions/InMemoryLogReplay.scala

```diff
@@ -72,8 +72,8 @@ class InMemoryLogReplay(
     }
   }
 
-  private def getTombstones: Iterable[FileAction] = {
-    tombstones.values.filter(_.delTimestamp > minFileRetentionTimestamp)
+  def getTombstones: scala.collection.mutable.HashMap[URI, RemoveFile] = {
```

Comment:
nit: it's better to return an immutable map

Reply:
makes sense. thanks!